### DISCUSSION TITLE
Importing PDF or Word files does nothing

### DISCUSSION THREAD

**Discussion by ckambiselis** [jgraph/drawio-desktop#1326] [Q&A]
I have been trying to find a way to import PDF files of schematics that you could search their text, but when I use Import from the file menu, the loading circle appears for a second and nothing happens, tried both PDF and Word files. Until now I have been exporting PNG files from Acrobat but the text can't be searched of course.

**Comment by davidjgraph** [comment]
We're not a PDF or word reader, we're a diagramming tool.
